pkg/driver/node_linux.go

```diff
@@ -48,12 +60,26 @@ func (d *nodeService) findDevicePath(devicePath, volumeID, partition string) (st
 	}
 
 	if exists {
```

Comment:
should we still try to fallback to the findNvmeVolume method in case there is an error in this block (if lstat or evalsymlinks somehow fails)? Is there any situation where we receive a devicePath that exists but that could not be lstatted or evalsymlink'd? If not then it is not necessary to fallback and this PR is good as-is

Reply:
@wongma7 I cannot think of a situation where that could be possible.